ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -1554,6 +1554,75 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi-ovn-ipv6,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Always run is a lot to add to this repo.  What does this job do better than using cluster-bot?

Reply:
This [upgrade test was added](https://github.com/openshift/origin/pull/26162) and it seems to panic on metal upgrades sometimes: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.9-e2e-metal-ipi-upgrade/1410486425048059904

I figured having on-prem upgrade coverage on origin makes sense,